example/zipkin_like/span.py

```diff
@@ -135,7 +146,7 @@ def is_debug(self):
         return self.trace_context.flags & DEBUG_FLAG == DEBUG_FLAG
 
     def __str__(self):
-        from .encoding import TraceContextEncoder
+        from .marshaling import TraceContextEncoder
```

Comment:
from an offline discussion w/ @bensigelman I thought we were going with...

  > Proposal:
%s/Marshaler/Encoder/g (etc)
%s/​Unmarshaler/Decoder/g (etc)
trace_context_(from|to)_(binary|text) in python, TraceContext(From|To)(Binary|Text) in golang